server/db.js

```diff
@@ -0,0 +1,22 @@
+const environment = process.env.NODE_ENV || 'development'
+const config = require('../../knexfile')[environment]
+const connection = require('knex')(config)
+
+module.exports = {
+  getMessages, createmessage, getPeople
+}
+
+function getMessages(connection) {
+  const db = conn || connection
+  return db('messages').select()
```

Comment:
`.select('sender_id as senderId ...`

And this should also be joined with `people`.